src/app/app.component.html

```diff
@@ -5,7 +5,7 @@ <h1 class="title is-1">
         Metacoin ~ Angular4 + Truffle Starter Dapp
       </h1>
       <h2 class="title">
-        You have <span class="is-medium has-underline">{{balance}}</span> META
+        You have <span class="is-medium has-underline">{{balance}} {{pong?.staticData.pongval}}</span> META
```

Comment:
Can we have one value here? Otherwise this shows up as two separate numbers on the UI

Reply:
Yup. It's one value. This is the reason why I create these package. All staticData is updated on ``` this._ethObservable.refresh();```